CHANGELOG.rst

```diff
@@ -18,6 +18,7 @@ Changed
 - Print info message when running Rasa X and a custom model server url was specified in ``endpoints.yml``
 - If a ``wait_time_between_pulls`` is configured for the model server in ``endpoints.yml``,
   this will be used instead of the default one when running Rasa X
+- Training Luis data with ``luis_schema_version`` higher than 4.x.x will show a warning instead of throwing an exception
```

Comment:
we've changed the changelog handling. please do not modify `CHANGELOG.rst`. instead, please create a new newsfile in `changelog/ `(more info https://github.com/RasaHQ/rasa/tree/master/changelog)